crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
+        }
+        // typo in println() function
+        prentln("Sum: $sum")
```

Comment:
Consider using `sum()` function for calculating the sum of elements in `largeList` for better performance.
```suggestion
// Inefficient way to calculate the sum of elements
    var sum = 0
    fun calculateSum() {
        sum = largeList.sum()
        // typo in println() function
        prentln("Sum: $sum")
        // Unnecessary object creation inside a loop
```